docs/resources/live_url_authentication.md

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "Live"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_live_url_authentication"
+description: |-
+  Manages an URL authentication resource within HuaweiCloud.
+---
+
+# huaweicloud_live_url_authentication
+
+Manages an URL authentication resource within HuaweiCloud.
+
+-> The current resource is a one-time resource, and destroying this resource will not change the current status.
+
+-> Before creating the resource, you need to cofiguration the URL validation first (creating
+  the URL validation resource first).
+  [documentation](https://support.huaweicloud.com/iLive-live/live_01_0049.html)
```

Comment:
Please use the English document link.

Reply:
-> Before creating the resource, you need to cofiguration the URL validation first (creating
  the URL validation resource first). Refer to [URL Validation](https://support.huaweicloud.com/intl/en-us/iLive-live/live_01_0049.html) for more details.